.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
I think it has to be .idea/ so that all the files are ignored

Reply:
Change made